server/sonar-server/src/test/java/org/sonar/server/issue/IssueUpdaterTest.java

```diff
@@ -139,7 +140,7 @@ public void verify_notification_on_branch() throws Exception {
     IssueChangeNotification issueChangeNotification = notificationArgumentCaptor.getValue();
     assertThat(issueChangeNotification.getFieldValue("key")).isEqualTo(issue.key());
     assertThat(issueChangeNotification.getFieldValue("projectKey")).isEqualTo(project.getDbKey());
-    assertThat(issueChangeNotification.getFieldValue("projectName")).isEqualTo(project.name());
+    assertThat(issueChangeNotification.getFieldValue("projectName")).isEqualTo(project.longName());
```

Comment:
I think that actually, the Unit Test was right here. `name()` should be used for a project, rather than `longName`.
In practice, `name()` and `longName()` have the same value for a project. But in theory, `longName()` does not make sense for a project.

The change should be in `IssueChangeNotification#setComponent`

Reply:
During merge after the branch feature there was some inconsistencies, so I kept the value from the merge. I'll update this to name().